Rubberduck.CodeAnalysis/Inspections/Concrete/EnumerationDeclaredWithinWorksheetInspection.cs

```diff
@@ -0,0 +1,35 @@
+﻿using Rubberduck.CodeAnalysis.Inspections.Abstract;
+using Rubberduck.Parsing.Symbols;
+using Rubberduck.Parsing.VBA;
+using Rubberduck.Parsing.VBA.DeclarationCaching;
+using Rubberduck.Resources.Inspections;
+using Rubberduck.VBEditor.ComManagement;
+using Rubberduck.VBEditor.SafeComWrappers;
+
+namespace Rubberduck.CodeAnalysis.Inspections.Concrete
+{
+    internal class PublicEnumerationDeclaredWithinWorksheetInspection : DeclarationInspectionBase
```

Comment:
The comment about the xml-doc should have been here.
In addition, the concrete inspectionsdhould be sealed.